.ci/azure-pipelines/env.yml

```diff
@@ -99,3 +99,48 @@ jobs:
       tags: "$(tag)"
       condition: and(eq(variables['Build.Repository.Name'], 'PointCloudLibrary/pcl'),
                      eq(variables['Build.SourceBranch'], 'refs/heads/master'))
+- job: BuildAndPushWindows
+  timeoutInMinutes: 360
+  displayName: "Env"
+  pool:
+    vmImage: 'windows-latest'
+  strategy:
+    matrix:
+      Winx86:
+        platform: x86
+        tag: winx86
+      Winx64:
+        platform: x64
+        tag: winx64
+  steps:
+  - task: Docker@2
+    displayName: "Build docker image"
+    inputs:
+      command: build
+      arguments: |
+        --no-cache
+        --build-arg platform=$(platform)
+        -t $(dockerHubID)/env:$(tag)
+      dockerfile: '$(Build.SourcesDirectory)/.dev/docker/windows/Dockerfile'
+      tags: "$(tag)"
+      volumes: $(Build.SourcesDirectory):/pcl
+      
+  - script: |
+      docker run --rm $(dockerHubID)/env:$(tag) \
```

Comment:
In ubuntu, we need to convert these statements into a string and pass it to bash. Some adaptation to windows might be needed:
* powershell is the default shell, set in Dockerfile
* mkdir might not be present
* cmake.exe instead of cmake?

Reply:
Yeah, you are also on to something. I think I'm close though 😁